WalletWasabi.Fluent/ViewModels/NavBar/NavBarViewModel.cs

```diff
@@ -43,6 +41,13 @@ public NavBarViewModel(UiContext uiContext)
 
 	public ReadOnlyObservableCollection<WalletPageViewModel> Wallets { get; }
 
+	// AutoInterfaces (such as IWalletModel) cannot be seen by AutoNotifyGenerator.
+	public IWalletModel? SelectedWalletModel
+	{
+		get => _selectedWalletModel;
+		set => this.RaiseAndSetIfChanged(ref _selectedWalletModel, value);
+	}
```

Comment:
You mentioned something about the sequential processing of source generators (I don't remember exactly). Is it not possible to solve this?

Reply:
Yes. This is the inverse case. `AutoInterface` depends on `AutoNotify`, so the latter runs first and its output is seen by the former, but not the other way around. 

We can't workaround this case unfortunately.